QUALITY_DECLARATION.md

```diff
@@ -1,132 +1,194 @@
-This document is a declaration of software quality for the `rcutils` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
 
-# `rcutils` Quality Declaration
-
-The package `rcutils` claims to be in the **Quality Level 1** category.
 
-Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004](https://www.ros.org/reps/rep-2004.html).
 
-## Version Policy
+This document is a declaration of software quality for the `rcutils` package, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
 
-### Version Scheme
+# `rcutils` Quality Declaration
 
-`rcutils` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning), and is at or above a stable version, i.e. `>= 1.0.0`.
+The package `rcutils` claims to be in the **Quality Level 4** category.
 
-### API Stability Within a Released ROS Distribution
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004 ](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#quality-level-1) of the ROS2 developer guide.
 
-`rcutils` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+## Version Policy [1]
 
-### ABI Stability Within a Released ROS Distribution
+### Version Scheme [1.i]
+`rcutils` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning)
 
-`rcutils` contains C code and therefore must be concerned with ABI stability, and will maintain ABI stability within a ROS distribution.
-
-### Public API Declaration
+### Version Stability [1.ii]
+Currently this package version is 0.7.5, soon to be released as 1.0.0
```

Comment:
How about keeping this declaration completely non-aspirational and removing all the "soon" statements? We know we're planning on getting to it in the coming months, so we can update the entire sentence when we do.

Also, this kind of statement tends to not age well in case we don't manage to get to it. Future readers won't know when "soon" is.

Reply:
Completed in [f14f23f](https://github.com/ros2/rcutils/pull/240/commits/f14f23f98d2a850a620d40e809f002227627eda1)